### PR TITLE
Bug 1575888 - Enable the What's New toolbar badge

### PR DESCRIPTION
**Pull request by piatra**
There's no risk of showing up accidentally because there are no what's new messages yet.

### CODE REVIEW

File: lib/OnboardingMessageProvider.jsm [hunk 1]
```diff
@@ -408,6 +409,28 @@ const ONBOARDING_MESSAGES = () => [
     },
     trigger: { id: "protectionsPanelOpen" },
   },
+  {
+    id: `WHATS_NEW_BADGE_${FIREFOX_VERSION}`,
+    template: "toolbar_badge",
+    content: {
+      delay: 5 * 3600 * 1000,
```
Comment (Mardak) [inline lib/OnboardingMessageProvider.jsm]: Should this be 5 minutes or 5 hours?

Comment (Mardak) [reply lib/OnboardingMessageProvider.jsm]: The comment was indeed previously `// delay: 5 * 3600 * 1000,`

**Comment by piatra** [comment]
Thanks for catching that. It should be 5 minutes.

**Comment by piatra** [comment]
I made it a little more obvious by defining a `cost ONE_MINUTE`

**Comment by Mardak** [comment]
This was reverted with f867e245f387e1e41999f958d24daf986faaf593 for accessing earliestFirefoxVersion -> UpdateManager -> active-update.xml

Backed out changeset 2aa1a97003cb (bug 1576284) for browser-chrome failures at browser/base/content/test/performance/browser_startup_mainthreadio.js

Backout: https://hg.mozilla.org/integration/autoland/rev/4f018843d990d49612c98196c451c30b288feb1a

Failure push: https://treeherder.mozilla.org/#/jobs?repo=autoland&revision=2aa1a97003cb26bc0fe0aa35f46ba979c70ca09a

Failure log: https://treeherder.mozilla.org/logviewer.html#/jobs?job_id=263295950&repo=autoland&lineNumber=2548

**Pull request merged**